build.gradle

```diff
@@ -46,69 +44,23 @@ subprojects {
         }
     }
 
-
     dependencies {
-        compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlin_version}"
-        compile "org.jetbrains.kotlin:kotlin-reflect:${kotlin_version}"
+        implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:${kotlin_version}"
+        implementation "org.jetbrains.kotlin:kotlin-reflect:${kotlin_version}"
 
         compile 'com.github.alexeykoren:konfig:2.0.4'
```

Comment:
Let's do fork to Luxoft repo, it's gonna to be more stable

Reply:
May be we should propose PR with our changes?

Reply:
Yes, let's just do a task in backlog

Reply:
https://github.com/hyperledger-labs/cordentity/projects/1#card-18735735